conf/edge.yaml

```diff
@@ -1,5 +1,7 @@
 mqtt:
-    server: tcp://127.0.0.1:1883
+    internal-mqtt: tcp://127.0.0.1:1884
```

Comment:
We are changing the APIs here, @Baoqiang-Zhang @qizha 

Any comments?

Reply:
edge.yaml is a config file for kubeedge, no APIs changing.  @m1093782566

Reply:
Configuration parameters is also API - it's API of binary file. We need to keep backward compatibility as well for component configuration and there should be a mechanism to deprecate old configuration. 

I am shocked to see the opinionated behavior change.

Reply:
A general comment: kubeedge is open sourced now and someone(I don't know the exact people though) probably is running it in their system, any change to user behavior should be cautious.